model/patch/patch.go

```diff
@@ -138,6 +138,7 @@ type GitMetadata struct {
 type Patch struct {
 	Id              mgobson.ObjectId       `bson:"_id,omitempty"`
 	Description     string                 `bson:"desc"`
+	Path            string                 `bson:"path"`
```

Comment:
we should make this omitempty, since we won't typically be providing it (this prevents us from explicitly storing the empty string in the mongoDB document)

Reply:
Changed to omitempty in latest commit